grid/migrations/versions/b2ef8dc4f38b_workers_table.py

```diff
@@ -0,0 +1,49 @@
+"""workers table
```

Comment:
do we want/ to keep this file?

Reply:
yes absolutely. This defines the schema for our persistence layer